src/components/BreadcrumbsItem/BreadcrumbsItem.module.css

```diff
@@ -102,7 +102,7 @@
  * Breadcrumbs Icon - a separator between breadcrumb links.
  */
 .breadcrumbs__separator {
-  color: var(--eds-theme-color-icon-neutral-subtle);
+  @apply text-icon-neutral-subtle-default;
```

Comment:
This will only work with Storybook and consumers with tailwind enabled with configs extending our config, so I think these kinds of change should be punted and combined with enabling TW with general EDS component use